x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_enable_section.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHealth,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiSpacer,
+  EuiSwitch,
+  EuiTitle,
+} from '@elastic/eui';
+import {
+  DETECTION_ENTITY_DASHBOARD,
+  RISKY_HOSTS_DOC_LINK,
+  RISKY_USERS_DOC_LINK,
+} from '../../../common/constants';
+import * as i18n from '../translations';
+
+const docsLinks = [
+  {
+    link: DETECTION_ENTITY_DASHBOARD,
+    label: i18n.EA_DOCS_DASHBOARD,
+  },
+  {
+    link: RISKY_HOSTS_DOC_LINK,
+    label: i18n.EA_DOCS_RISK_HOSTS,
+  },
+  {
+    link: RISKY_USERS_DOC_LINK,
+    label: i18n.EA_DOCS_RISK_USERS,
+  },
+];
+
+export const RiskScoreEnableSection = () => {
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <>
+      <>
+        <EuiTitle>
+          <h2>{i18n.RISK_SCORE_MODULE_STATUS}</h2>
+        </EuiTitle>
+        <EuiSpacer size="m" />
+        <EuiFlexItem grow={0}>
+          <EuiHorizontalRule margin="s" />
+          <EuiFlexGroup justifyContent="spaceBetween">
+            <EuiFlexItem grow={false}>{i18n.ENTITY_RISK_SCORING}</EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup gutterSize="s" alignItems={'center'}>
+                <EuiFlexItem css={{ minWidth: '50px' }}>
```

Comment:
Magic number? What does this correspond to?

Reply:
To prevent label of switch from, moving